models/articles-models.js

```diff
@@ -1,15 +1,84 @@
 const db = require("../db/connection.js");
 
-fetchArticles = () => {
-  const queryString = `
+fetchArticles = (topic, sort_by, order) => {
+  const validTopicByOptions = ["mitch", "cats", "paper"];
```

Comment:
Can't make out quite what some of this code for Promise.rejects purpose is. The validOptions arrays checks looks logical but unsure why we're doing number type checks with isNan and parseInt? Lets have a chat after the lecture as I suspect this can be simplified.